### DISCUSSION TITLE
Plugin - Filter multicast ip in DynamicModelChoiceField

### DISCUSSION THREAD

**Discussion by ChrisdAutume** [netbox-community/netbox#10861] [Q&A] [answered]
Hi there,

I looking to filter only multicast ip in a Form using a DynamicModelChoiceField.
Unfortunately, I'm not succeeding using query_params unless using tag (but tags isn't a good solution).

Do someone have an idea ?

**Comment by candlerb** [comment, marked as answer]
Try filtering on `parent=224.0.0.0/4`. This just checks whether the IP address is contained within the given block; you do not need to have an actual Prefix object in the database.

See:
* https://github.com/netbox-community/netbox/blob/v3.3.7/netbox/ipam/filtersets.py#L469-L472
* https://github.com/netbox-community/netbox/blob/v3.3.7/netbox/ipam/filtersets.py#L575-L585

**Reply by ChrisdAutume** [reply]
amazing.

I missed that.

Thanks
